lib/quickbooks/model/base_model.rb

```diff
@@ -2,6 +2,7 @@ module Quickbooks
   module Model
     class BaseModel
       include Definition
+      include ActiveModel::AttributeMethods
```

Comment:
I had to add this because the tests were broken 🤔 There's a PR about that: https://github.com/ruckus/quickbooks-ruby/pull/453/files